meerkat_frontend/src/js/technical/clinics.js

```diff
@@ -148,21 +175,45 @@ function buildClinicsReturnVisits(clinicName, weeks) {
     });
 }
 
-//This function will get the completness value and put it in the right position in the array ...
-function buildClinicsCompleteness(clinicName, weeks) {
+function buildClinicsCD(clinicName, weeks) {
     var currentWeek = week;
-    var counter = 2; // i need the last 3 values only which represent the last 3 weeks ...
     $.each(clinicDataArray, function(index, value) {
         if (value.clinicName.toString() === clinicName.toString()) {
-            for (i = currentWeek - 2; i < currentWeek; i++) {
-                var itemName = i + "C";
-                value[itemName] = calculateCompleteness(weeks[weeks.length - counter]) + ' % ';
-                counter = counter - 1;
+            for (i = currentWeek - 2; i <= currentWeek; i++) {
+                var itemName = i + "CD";
+                value[itemName] = weeks[i];
             }
         }
     });
 }
 
+function buildClinicsNCD(clinicName, weeks) {
+    var currentWeek = week;
+    $.each(clinicDataArray, function(index, value) {
+        if (value.clinicName.toString() === clinicName.toString()) {
+            for (i = currentWeek - 2; i <= currentWeek; i++) {
+                var itemName = i + "NCD";
+                value[itemName] = weeks[i];
+            }
+        }
+    });
+}
+
+//This function will get the completness value and put it in the right position in the array ...
```

Comment:
@ZiadFra Probably a good idea to avoid commenting out bits of code like this unless it specifically exists as a clear reminder to other developers.  Just remove it, we can always fetch it back out of the git tree if you decide you need it back.